flask_resty/pagination.py

```diff
@@ -184,32 +189,16 @@ class CursorPaginationBase(LimitPagination):
     cursor_arg = 'cursor'
 
     def get_column_orderings(self, query, view):
-        column_orderings = tuple(
-            self.get_column_ordering(expression)
-            for expression in query._order_by
-        )
+        column_orderings = view.sorting.get_request_column_orderings(view)
```

Comment:
so you are making it necessary for a view with pagination to have sorting, which is a breaking change. And might not be "strictly" necessary because the ordering might be defined in the model (not saying is a good idea, but it's allowed)